configure.cmd

```diff
@@ -240,7 +241,12 @@ if "%NEW_STYLE_BUILD%"=="0" (
 if "%BUILD_ENVIRONMENT%" == "MinGW" (
     cmake -G %CMAKE_GENERATOR% -DENABLE_CCACHE:BOOL=0 -DCMAKE_TOOLCHAIN_FILE:FILEPATH=%MINGW_TOOCHAIN_FILE% -DARCH:STRING=%ARCH% %BUILD_TOOLS_FLAG% %* "%REACTOS_SOURCE_DIR%"
 ) else (
-    cmake -G %CMAKE_GENERATOR% -DCMAKE_TOOLCHAIN_FILE:FILEPATH=toolchain-msvc.cmake -DARCH:STRING=%ARCH% %BUILD_TOOLS_FLAG% -DRUNTIME_CHECKS:BOOL=%VS_RUNTIME_CHECKS% %* "%REACTOS_SOURCE_DIR%"
+    if %USE_CLANG_CL% == 1 (
```

Comment:
else if?
other than that, lgtm.

Reply:
My batch-fu is not top notch so improvements are most welcome :)

Reply:
```
if "%BUILD_ENVIRONMENT%" == "MinGW" (
) else if %USE_CLANG_CL% == 1 (
) else (
)
```